docs/src/main/tut/typeclasses.md

```diff
@@ -37,6 +36,14 @@ implicit val stringShow = new Show[String] {
 log("a string")
 ```
 
+This example demonstrates a powerful property of the typeclass
+pattern, which distinguishes it from a Java interface, which is that
```

Comment:
What do you think about changing this to something like "which distinguishes it from a Java interface that you would make your custom class extend".

My wording isn't great, but here's what's going through my head: often when I'm explaining type classes to someone coming from a Java background, I describe them as being like `Comparator` as opposed to `Comparable`. For people familiar with those two interfaces, this analogy often seems to help them grasp the concept quickly. You can do type classes in Java; it's just very verbose.